src/components/AddressField.tsx

```diff
@@ -0,0 +1,56 @@
+
+import React, { useState } from 'react'
+import { TextField } from '@mui/material'
+import Resolver from '@rsksmart/rns-resolver.js'
+
+// @ts-ignore
+// eslint-disable-next-line new-cap
+const resolver = new Resolver.forRskTestnet()
```

Comment:
We should pass this as parameter of the component

Reply:
true, on it

Reply:
Done